xmls/NWA12/CEFI_NWA12_cobalt.xml

```diff
@@ -94,7 +95,7 @@ frecheck -r layout -p ncrc5.intel22 -x CEFI_NWA12_cobalt.xml CEFI_NWA12_COBALT_V
   <property name="STATIC" value=""/>
 
   <!-- Path to world-shared location with common NWA12 input files -->
-  <property name="NWA12_WORLD" value="/gpfs/f5/gfdl_med/world-shared/northwest_atlantic/nwa12_input"/>
+  <property name="NWA12_WORLD" value="/gpfs/f6/ira-cefi/world-shared/northwest_atlantic/nwa12_input"/>
```

Comment:
What do you think about putting properties like this with the platforms.xml, so that it can point to f5 for c5 platforms and f6 for c6? 

That or where we have `<dataSource>`, I think something like `<dataSource platform='ncrc6.intel23'>` would work

Reply:
Thanks for the suggestion, @andrew-c-ross. I moved the data folder properties to the platforms.xml now.